include/eve/module/core/regular/swap_adjacent_groups.hpp

```diff
@@ -13,6 +13,55 @@
 
 namespace eve
 {
+//================================================================================================
+//! @addtogroup core_bitops
```

Comment:
This is a shuffle. I thought we have a group for shuffles. Definetly not a bitop